src/atlasdcat/mapper.py

```diff
@@ -221,6 +232,21 @@ def _generate_name(value: str) -> str:
     return value.replace(" ", "").lower()
 
 
+def _timestamp_to_date(ts: float):
+    """Convert timestamp to date.
+
+    Args:
+        ts: Timestamp in seconds or milliseconds
+
+    Returns:
+        Date
+    """
+    if ts > time.time():
+        return dt.datetime.fromtimestamp(ts/1000.0).strftime('%Y-%m-%d')
+    else:
+        return dt.datetime.fromtimestamp(ts).strftime('%Y-%m-%d')
```

Comment:
Mulig å slippe duplisering av all logikken, men bare koble deling på 1000 til if'en?
```suggestion
    if ts > time.time():
        ts /= 1000.0

    return dt.datetime.fromtimestamp(ts).strftime('%Y-%m-%d')
```